tools/maven/checkstyle.xml

```diff
@@ -25,7 +25,7 @@
   <module name="TreeWalker">
     <property name="tabWidth" value="2"/>
     <module name="LineLength">
-      <property name="max" value="120"/>
+      <property name="max" value="130"/>
```

Comment:
@tkakantousis in Hopsworks we have 120, should we be consistent?

Reply:
Yes would be good to do so.
@NetsanetGeb what was the reason for changing the checkstyle?

Reply:
@tkakantousis,   I changed the checkstyle because we added some Hudi parameters into some of the methods in Hops-util  which in turn maximized the line length.